charts/argo/templates/server-deployment.yaml

```diff
@@ -31,6 +31,7 @@ spec:
         - name: argo-server
           args:
           - server
+          - --configmap={{ .Release.Name }}-{{ .Values.controller.name }}-configmap
```

Comment:
Is this intended to be part of this PR? also https://github.com/argoproj/argo-helm/pull/271 should allow you to set this using `extraArgs` now if desired.

Reply:
Yes, there is a configmap in the chart that is not using the default configmap name. First versions of 2.7.x didn't even start without this option because Argo couldn't find the configmap.

Reply:
Yup makes sense, best be sensible by default.